linetools/analysis/voigt.py

```diff
@@ -176,7 +176,7 @@ def voigt_from_abslines(iwave, line, fwhm=None, ret=['vmodel'],
         if const.c.to('km/s')*dwave/medwave > minb/10.:
             wmin = np.min(iwave.to('AA').value)
             wmax = np.max(iwave.to('AA').value)
-            nsub = int(np.round( (np.log10(wmax)- np.log10(wmin)) / 1.449E-6)) + 1
+            nsub = int(np.round( (np.log10(wmax)- np.log10(wmin)) / 1.449E-6)) + 2
```

Comment:
Why is this needed to be + 2 now? Could you add a inline comment about it?

Reply:
Somehow +1 didn't cover beyond wmax.  It's not 100% necessary
with the extrapolate in rebin, but on the safe side.  Small comment added.